apps/web/src/services/cost-explorer/cost-analysis/modules/CostAnalysisDataTable.vue

```diff
@@ -306,21 +310,18 @@ const handleChange = async (options: any = {}) => {
     if (costAnalysisPageState.period) tableState.items = getRefinedChartTableData(results, costAnalysisPageState.granularity, costAnalysisPageState.period);
     tableState.more = more;
 };
+const costAnalyzeExportApiQueryHelper = new ApiQueryHelper();
```

Comment:
nitpick: Please use QueryHelper instead of ApiQueryHelper in case that you don't need any other options like pagination or sort. 
It's pretty risky though...